public/stylesheets/style.css

```diff
@@ -0,0 +1,21 @@
+body {
```

Comment:
This should be a scss file. Convert css to scss and create a scss task in gulp to compile scss to css

Reply:
done